README.md

```diff
@@ -42,17 +42,6 @@ ARGUMENTS
 ---------
 <!-- TODO replace this with output from command line --help -->
 
---deposits-dir | -d (optional)
```

Comment:
Why remove these arguments? According to line 10 you still have these arguments...